tools/lkl/tests/net.sh

```diff
@@ -28,6 +28,11 @@ cleanup_backend()
 
 get_test_ip()
 {
+    # skip if we don't explicitly allow testing on live host network
+    if [ -z "$LKL_TEST_ON_LIVE_HOST_NETWORK" ]; then
```

Comment:
Where is this variable configured?

Reply:
By default it is not set, the user should set it if they want to run this test. I run into issues on more complicated network setups - the test would break connectivity of the host, so I think we should not run it anymore by default.

Also these tests are already skipped in CI due to missing privileges.

Reply:
Thanks, make sense to me. 

IIRC, we can’t use ping from LKL to external hosts on GitHub action runner (probably due to  azure design).